File: plugins/OrchidPosts/src/main/java/com/eden/orchid/posts/PostsPaginator.java
package com.eden.orchid.posts;

import com.eden.orchid.api.options.Option;
import com.eden.orchid.api.options.OptionsHolder;
import com.eden.orchid.api.options.annotations.IntDefault;

public class PostsPaginator implements OptionsHolder {

    @Option
    @IntDefault(100)
    public int pageSize;

}

File: plugins/OrchidPosts/src/main/kotlin/com/eden/orchid/posts/model/PostsPaginator.kt
package com.eden.orchid.posts.model

import com.eden.orchid.api.options.OptionsHolder
import com.eden.orchid.api.options.annotations.IntDefault
import com.eden.orchid.api.options.annotations.Option


class PostsPaginator : OptionsHolder {

    @Option
    @IntDefault(100)
    var pageSize: Int = 0

    @Option
    lateinit var permalink: String

}

